instance/create_instance.py

```diff
@@ -0,0 +1,93 @@
+from libcloud.compute.types import Provider
+from libcloud.compute.providers import get_driver
+import os
+import time
+import argparse
+
+# Parse flag arguments
+
+parser = argparse.ArgumentParser()
+
+def add_flag(flag_name, default):
+    parser.add_argument(f'--{flag_name}', dest=flag_name,
+                        action='store', default=default)
+
+# New instance info
+add_flag('branch', 'aosp-master')
+add_flag('target' ,'aosp_cf_x86_phone-userdebug')
+add_flag('user_id', '00001')
+add_flag('zone', 'us-central1-b')
+parser.add_argument('--tag', action='append', dest='tags', default=[])
+
+# GCE Project info
+add_flag('datacenter', 'us-central1-b')
+add_flag('project', 'cloud-android-testing')
+
+# Signaling server info
+add_flag('sig_server_addr', '127.0.0.1')
+add_flag('sig_server_port', '8443')
+
+
+args = parser.parse_args()
+
+
+# Get GCE Driver
+ComputeEngine = get_driver(Provider.GCE)
+driver = ComputeEngine('', '',
+                       datacenter=args.datacenter, project=args.project)
+
+
+def wait_for_instance(instance_name):
+    not_running = 1
+    while not_running != 0:
+        time.sleep(5)
+        # uptime returns a value other than 0 when not successful
+        not_running = os.system(f'gcloud compute ssh {instance_name} --zone={args.zone} -- uptime')
+
+def fatal_error(msg):
+    print(f'Error: {msg}')
+    exit()
```

Comment:
Could you maybe do a follow-up PR to pull these and other re-usable methods into a shared lib to avoid code duplication?

Reply:
Yeah I'm already doing that in the next PR! 👍